compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/MouseWheelScrollable.kt

```diff
@@ -211,6 +224,7 @@ internal class MouseWheelScrollNode(
             return
         }
         var animationState = AnimationState(0f)
+        var waitNextEventTimeMs = 0
```

Comment:
`dispatchMouseWheelScroll` becomes complex and needs refactoring (and tests) in the future. There are ways to make it more understandable with less bugs by reducing/combining states and reading them more closely to where they are written.

Reply:
Simplifying/refactoring is always welcome